jest-setup.ts

```diff
@@ -102,4 +103,22 @@ function parseUrl(
   };
 }
 
+if (typeof window.URL.createObjectURL === 'undefined') {
+  window.URL.createObjectURL = (): string => 'kjkj';
```

Comment:
`kjkj`?

Reply:
Was just a random string for this mocking. I have changed the string in hope that it clarifies it better.